caracara/common/policy_wrapper.py

```diff
@@ -281,6 +283,10 @@ class Policy:
     settings_groups: List[PolicySettingGroup] = None
     settings_key_name = "settings"
 
+    # TODO: Replace with a proper wrapper around ioa_rule_groups, this is temporary so that this
+    #       result is still reachable from the 'dump' method when style="prevention".
+    _raw_ioa_rule_groups: dict = None
```

Comment:
What data types does the dictionary map? Can we do something like this?
```_raw_ioa_rule_groups: Dict[str, str] = None```

Reply:
Now that I'm looking at the documentation (ctrl+f for `responses.PreventionPolicyV1` [here](https://assets.falcon.crowdstrike.com/support/api/swagger.html)), I meant to put something like `List[dict]`. Even still though, `List[Dict[str, str]]` wouldn't be right because, looking at the docs (`responses.IOARuleGroupV1`), the value type could be `str`, `bool`, or `int`. I guess I could do `List[Dict[str, Union[str, bool, int]]]`, but if I'm going that far I might as well use a `TypedDict` and specify the whole schema. Maybe we could do `List[Dict[str, Any]]` as a good middle ground?

Reply:
```List[Dict[str, Any]]``` is a perfect middle ground :)